src/inmanta/server/services/orchestrationservice.py

```diff
@@ -814,9 +820,163 @@ async def _put_version(
             all_agents: set[str] = {res.agent for res in rid_to_resource.values()}
             all_agents.add(const.AGENT_SCHEDULER_ID)
 
+            type_to_agent: dict[str, set[str]] = defaultdict(set)
+            for res in rid_to_resource.values():
+                type_to_agent[str(res.resource_type)].add(res.agent)
```

Comment:
I wonder if we should have the exporter produce this? i.e. there is no reason to talk about types here, it is all about modules?

Reply:
refactored into the exporter